tests/base_tests.py

```diff
@@ -182,6 +182,15 @@ def get_user(username: str) -> ab_models.User:
         )
         return user
 
+    @staticmethod
+    def get_role(name: str) -> ab_models.User:
```

Comment:
If we're returning `one_or_none()`, this should be optional:
```suggestion
    def get_role(name: str) -> Optional[ab_models.User]:
```